### PR TITLE
[Channels] Add matrix_id to get_chan

### PR DESCRIPTION
**Pull request by Herklos**
Depends on https://github.com/Drakkar-Software/OctoBot-Evaluators/pull/51
Depends on https://github.com/Drakkar-Software/OctoBot-Channels/pull/33

### CODE REVIEW

File: octobot_evaluators/api/evaluators.py [hunk 1]
```diff
@@ -85,6 +86,11 @@ async def stop_evaluator(evaluator) -> None:
     return await evaluator.stop()
 
 
+async def stop_evaluators_channel(matrix_id) -> None:
+    # TODO
```
Comment (GuillaumeDSM) [inline octobot_evaluators/api/evaluators.py:90]: what's the issue to implement it now ?

File: octobot_evaluators/channels/evaluator_channel.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    def __init__(self, matrix_id):
+        super().__init__()
+        self.matrix_id = matrix_id
+
+
+def set_chan(chan, name) -> None:
+    chan_name = chan.get_name() if name else name
+
+    try:
+        evaluator_chan = ChannelInstances.instance().channels[chan.matrix_id]
```
Comment (GuillaumeDSM) [inline octobot_evaluators/channels/evaluator_channel.py:46]: 👍

File: octobot_evaluators/api/evaluators.py (hunk 1 above)
Comment (Herklos) [reply octobot_evaluators/api/evaluators.py]: You commented 2min too early :smile:

File: octobot_evaluators/api/evaluators.py [hunk 3]
```diff
@@ -86,9 +89,16 @@ async def stop_evaluator(evaluator) -> None:
...
-    # TODO
-    pass
+async def stop_evaluator_channel(matrix_id, chan_name) -> None:
+    try:
+        await get_chan(chan_name, matrix_id).stop()
+    except Exception as e:
+        get_logger().exception(e, True, f"Error when stopping evaluator channel {chan_name}: {e}")
+
+
+async def stop_all_evaluator_channel(matrix_id) -> None:
```
Comment (GuillaumeDSM) [inline octobot_evaluators/api/evaluators.py:99]: I think we should name this method ```stop_all_evaluator_channels```, it would be clearer

**Pull request closed without merging**